frontend/src/routes/Upload.tsx

```diff
@@ -1039,7 +1057,7 @@ const finishUpload = async (
         }
 
         // Add ACL
-        {
+        if (!CONFIG.lockAclToSeries || !metadata.series?.acl) {
```

Comment:
Here you also check whether the series even has ACL, but above you don't. I guess this condition should exactly match the one deciding whether the ACL UI is locked.